projects/addon-editor/directives/tiptap-editor/tiptap-editor.directive.ts

```diff
@@ -42,6 +42,12 @@ export class TuiTiptapEditorDirective {
     @Output()
     readonly stateChange = this.editor.stateChange$;
 
+    @Output()
+    readonly focusChange = this.editor.focusChange$;
```

Comment:
Why do we need those? `focusin`/`focusout` events bubble. Also they are definitely not `change` if nothing changes :D We typically have `focusChange: Observable<boolean>` because it watches for focus and that state actually changes.

Reply:
fixed